Marlin/src/lcd/menu/menu_main.cpp

```diff
@@ -147,8 +158,8 @@ void menu_main() {
       }
     #endif // !HAS_ENCODER_WHEEL && SDSUPPORT
 
-    #if ENABLED(SDSUPPORT) || defined(ACTION_ON_RESUME)
-      #if ENABLED(SDSUPPORT)
+    #if ENABLED(SDSUPPORT) || ENABLED(HOST_ACTION_COMMANDS)
+      #if DISABLED(HOST_ACTION_COMMANDS)
         if (card.isFileOpen() && card.isPaused())
       #endif
           MENU_ITEM(function, MSG_RESUME_PRINT, lcd_resume);
```

Comment:
@InsanityAutomation — This needs a proper condition so it only appears when there is a print actually in progress. Other code looks at the print job timer and/or the `planner.movesplanned` value to tell that there is a print in progress. If the job timer is paused, then we surely know the print is paused. And if there are moves in the planner, we may act as though the print is in progress (not paused).

Reply:
Its inside the else of if(busy) so that should have it covered.